dev-scripts/make-release.sh

```diff
@@ -62,6 +61,16 @@ if [ -f "$INSTALLER_TARBALL" ]; then
     rm "$INSTALLER_TARBALL"
 fi
 
+# Insert the install scripts
+# Start by adding the base install command
+cp "$INSTALLER_SCRIPTS_DIR/install_ipfix-rita.sh" "$INSTALLER_DIR"
+sed -i "s|INSTALLATION_DIR=\"REPLACE_WITH_INSTALL_DIR\"|INSTALLATION_DIR=\"$INSTALLATION_DIR\"|g" $INSTALLER_DIR/install_ipfix-rita.sh
+sed -i "s|INSTALLATION_ETC_DIR=\"REPLACE_WITH_ETC_DIR\"|INSTALLATION_ETC_DIR=\"$INSTALLATION_ETC_DIR\"|g" $INSTALLER_DIR/install_ipfix-rita.sh
+sed -i "s|DOCKER_IMAGES=\"./REPLACE_WITH_TARBALL\"|DOCKER_IMAGES=\"$DOCKER_IMAGE_OUT\"|g" $INSTALLER_DIR/install_ipfix-rita.sh
```

Comment:
If these are the only variables we need to set in the install_ipfix-rita.sh script, I'm okay with using sed. 
If this grows, we may want to turn to something like [envsubst](https://linux.die.net/man/1/envsubst)